apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.ts

```diff
@@ -105,6 +108,8 @@ router.post(
         promptGeneral: req.body.prompt,
         promptUserInput: req.body.prompt_user_input,
         promptGrading: req.body.prompt_grading,
+        userId: res.locals.user.user_id,
```

Comment:
Per our discussion a week or two ago, we always want to track the _authenticated_ user here. That means reading from `res.locals.authn_user.user_id`.

Reply:
changed in 8f91ed5